quadrat/sass/blocks/_query.scss

```diff
@@ -2,4 +2,11 @@
 	.wp-block-post-title {
 		font-size: var(--wp--custom--heading--h3--font-size);
 	}
+}
+
+// If a query block is a direct child of a container, it some padding on mobile
+div[class^="wp-container-"] > .wp-block-query-loop {
+	@media screen and ( max-width: 704px ){ // This is the default content width + padding
```

Comment:
This value worries me since in the future the padding might change and this will not be the case any more

Reply:
Another issue that comes to mind is that this won't work for pages like 404, search etc where there's no loop